dts/common/nordic/nrf52840_partition.dtsi

```diff
@@ -4,8 +4,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/ {
+ / {
```

Comment:
indent should go back to how it was

Reply:
sorry I'm undoing a broken merge conflict...

Reply:
fixed